### PR TITLE
HotFix -Made slight improvements for cleaner routes

### PR DESCRIPTION
**Pull request by BerryGIT-ME**
### What this PR does (required):
- Made some slight improvements for cleaner routes

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -27,36 +27,19 @@ function App(): JSX.Element {
...
+                  <Switch>
+                    <Route exact path="/login" component={Login} />
+                    <Route exact path="/signup" component={Signup} />
+                    <Route exact path="/booking" component={Booking} />
+                    <Route exact path="/home" component={HomePage} />
+                    <Route exact path="/profile/:id?" component={Profile} />
+                    <Route path="/profile-listings" component={ProfileListings} />
+                    <ProtectedRoute exact path="/dashboard" component={Dashboard} />
+                    <ProtectedRoute exact path="/edit-profilel" component={EditProfile} />
+                    <Route path="*" render={() => <Redirect to="/login" />} />
```
Comment (ezeilo-su) [inline client/src/App.tsx:40]: :+1:

**Review by ezeilo-su — COMMENTED** [review]
Good work refactoring your code. But you forgot to link the issue here.

**Comment by rajivtitus** [comment]
@ezeilo-su This looks like a hotfix to make minor changes to existing code (it will not have any relevant issue) 😄

**Review by rajivtitus — APPROVED** [review]
LGTM 👍🏽

**Pull request merged**